test/e2e/egress_test.go

```diff
@@ -333,7 +333,7 @@ func testEgressClientIPFromVLANSubnet(t *testing.T, data *TestData) {
 			// By default, Pod will be SNATed to Node IP.
 			defaultClientIP := workerNodeIPv4(1)
 			if utilnet.IsIPv6String(tt.serverIP) {
-				serverIPStr = fmt.Sprintf("[%s]", tt.serverIP)
+				serverIPStr = fmt.Sprintf("%s", tt.serverIP)
```

Comment:
This line becomes useless as it actually does nothing. Please just remove it and replace all usage of `serverIPStr` with `tt.serverIP`